packages/requires.txt

```diff
@@ -1,7 +1,6 @@
 idna>=3.3,<4
 typer>=0.4.1,<1
 fastapi>=0.109.1,<1
-fastapi_websocket_pubsub==0.3.7
```

Comment:
This will be fixed to `0.3.9` once we release `fastapi_websocket_pubsub` with that fix - https://github.com/permitio/fastapi_websocket_pubsub/pull/82/files